### ISSUE TITLE
What is the unit of virials❓ [QUESTION]

### ISSUE THREAD

**Issue by YuanbinLiu** [mir-group/nequip#404]
I'd like to train virials as well. Could you point out what unit is used for the virial fitting?

**Comment by YuanbinLiu** [comment]
Sorry for the unclear description. I want to train on energy, forces, and virials (not stresses). I need to know which unit I have to use for fitting virials so that I can adjust them to the suitable format.

**Comment by Linux-cpp-lisp** [comment]
Aha, I see:  we do not do any unit conversions for either virials or stress, since `nequip` has no formal sense of units and just inherits the units of the dataset. So presumably eV in this case. 

See for some more details comments throughout: https://github.com/mir-group/nequip/blob/develop/nequip/nn/_grad_output.py#L346-L351

**Comment by rbjiawen** [comment]
Hi, @Linux-cpp-lisp .  In my recent time I have found that allegro is not accurate in predicting stress with eV/A3 units due to its very small value and I wonder if I am using the units incorrectly. As you mentioned, there is no formal sense of unit in allegro (https://github.com/mir-group/nequip/issues/404). So,
If I use ZBL with unit of metal and am training a model that can run MD with metal units using lammps, should I use eV for energy, eV/A for forces and Bars for Stress?

**Comment by rbjiawen** [comment]
Hi, @Linux-cpp-lisp . In my recent time I have found that allegro is not accurate in predicting stress with eV/A3 units due to its very small value and I wonder if I am using the units incorrectly. As you mentioned, there is no formal sense of unit in allegro (https://github.com/mir-group/nequip/issues/404). So,
If I use ZBL with unit of metal and am training a model that can run MD with metal units using lammps, should I use eV for energy, eV/A for forces and Bars for Stress in my training dataset? and do the weights in the loss functions need to be rebalanced?
